docs/html/user_guide.rst

```diff
@@ -772,6 +772,172 @@ archives are built with identical packages.
+   (e.g. ``pip install "packageA>0.44.*" "packageB>4.0.0"``)
+-  Asking pip to install *any* version of ``packageA`` and ``packageB``
+   by removing the version specifiers altogether (e.g.
+   ``pip install packageA packageB``)
+
+In the second case, pip will automatically find a version of both
+``packageA`` and ``packageB`` that depend on the same version of
+``packageC``, installing:
+
+-  ``packageA 0.46.0b0``, which depends on ``packageC 2.6.1``
+-  ``packageB 4.3.0`` which *also* depends on ``packageC 2.6.1``
+
+If you want to prioritise one package over another, you can add version
+specifiers to *only* the more important package:
+
+``pip install packageA==0.44.1b0 packageB``
+
+This will result in: - ``packageA 0.44.1b0``, which depends on
+``packageC 2.6.1`` - ``packageB 4.1.3`` which also depends on
+``packageC 2.6.1``
+
+Now that you have resolved the issue, you can repin the compatible
+package versions as required.
+
+3. Loosen the requirements of your downstream dependencies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Assuming that you cannot resolve the conflict by loosening the version
+of the package you require (as above), you can try to fix the issue on
+your *downstream dependency* by:
+
+-  Requesting that the package maintainers loosen *their* dependencies
+-  Forking the package and loosening the dependencies yourself
+
+Warning: If you choose to fork the package yourself, you are *opting
+out* of any support provided by the package maintainers. Proceed at your
+own risk!
+
+4. All requirements are loose, but a solution does not exist
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes it's simply impossible to find a combination of package
+versions that do not conflict. Welcome to `dependency
+hell <https://en.wikipedia.org/wiki/Dependency_hell>`__.
+
+In this situation, you could consider:
+
+-  Using an alternative package, if that is acceptable for your project.
+   See `Awesome Python <https://python.libhunt.com/>`__ for similar
+   packages.
+-  Refactoring your project to reduce the number of dependencies (for
+   example, by breaking up a monolithic code base into smaller pieces)
+
+Getting help
+------------
+
+If none of the suggestions above work for you, we recommend that you ask
+for help on:
+
+-  `Python user discuss <https://discuss.python.org/c/users/7>`__
+-  `Python user forums <https://www.python.org/community/forums/>`__
+-  `Python developers Slack channel <https://pythondev.slack.com/>`__
+-  `Python IRC <https://www.python.org/community/irc/>`__
+-  `Stack
+   Overflow <https://stackoverflow.com/questions/tagged/python>`__
+
+See `"How do I ask a good
+question?" <https://stackoverflow.com/help/how-to-ask>`__ for tips on
+asking for help.
+
+Unfortunately, **the pip team cannot provide support for individual
+dependency conflict errors**. Please *only* open a ticket on the `pip
+issue tracker <https://github.com/pypa/pip/issues>`__ if you believe
```

Comment:
To meet the width limit, I think [separated link and target](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links) might be prettier and easier to read as plain text.  Also I notice the extra underscores at the end.

Reply:
The two underscores make this an [anonymous link](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#anonymous-hyperlinks), which is the right thing to do here if the link is inlined. But I agree the separated link and target form would be better here (and in the bullet points above as well).

Reply:
Thanks, TIL